exchange_server/datadog_checks/exchange_server/data/conf.yaml.example

```diff
@@ -23,4 +27,31 @@ instances:
     ##               rate, histogram or counter
     #
     # additional_metrics:
-    #   - ['MSExchange Content Filter Agent', none, 'Messages that Bypassed Scanning', exchange.content_filter.bypassed_messages, gauge]
+    #   - [MSExchange Content Filter Agent, none, Messages that Bypassed Scanning, exchange.content_filter.bypassed_messages,
```

Comment:
Anyway to keep that on one line or instead have each item on a new line?

Reply:
I tried to do separate lines but after running `ddev validate config --sync` the `conf.yaml` file still automatically generates with the current output.

Reply:
Yeah I think we should update ddev to allow longer lines in that case. Even if we have to use a new option for that.
But that's for another time